src/mtool/api/test/Mtool_Env-Git-server.postman_environment.json

```diff
@@ -12,6 +12,16 @@
 			"value": "5000",
 			"enabled": true
 		},
+		{
+			"key": "telemetryIP",
+			"value": "107.108.83.97",
```

Comment:
Please change the IP to generic. Maybe you can use 127.0.0.1

Reply:
Thanks for the comment will change it